README.md

```diff
@@ -17,6 +17,17 @@ Download the latest releases from the [GitHub release page](https://github.com/j
 
 Pull the [jq image](https://github.com/jqlang/jq/pkgs/container/jq) to start quickly with Docker.
 
+
+#### Run with Docker
+##### Example: Extracting the version from a `package.json` file
+```bash
+docker run --rm -i ghcr.io/jqlang/jq:latest < package.json '.version'
+```
+##### Example: Extracting the version from a `package.json` file with a mounted volume
+```bash
+docker run --rm -i -v "${PWD}":/pwd -w /pwd ghcr.io/jqlang/jq:latest '.version' package.json
```

Comment:
One possible improvement could be to do `-v "$PWD:$PWD" -w "$PWD"` to make paths work more as expected

Reply:
Not sure I understand, Can you maybe share an example command that will act in an unexpected way?

Reply:
If one would happen to use absolute host paths in arguments etc they won't work, ex:
```sh
docker run --rm -i -v "${PWD}":/pwd -w /pwd ghcr.io/jqlang/jq:latest -f $PWD/file.jq ...
```
but even with the -v -w trick above paths outside host $PWD will not work... so not perfect either.

Reply:
I think this is a slight difference no need to fix. `-f` accepts relative path, `env.PWD` is different, but should be ok.

Reply:
Opened a PR https://github.com/jqlang/jq/pull/3105

Reply:
@itchyny yeap not big thing but it has hit me a couple of times before :)